lib/WP_Auth0_Lock10_Options.php

```diff
@@ -33,9 +33,8 @@ public function get_lock_show_method() {
   }
 
   public function get_code_callback_url() {
-    $protocol = $this->_get_boolean( $this->wp_options->get( 'force_https_callback' ) ) ? 'https' : null;
-
-    return site_url( 'index.php?auth0=1', $protocol );
+    $protocol = $this->_get_boolean( $this->wp_options->get( 'force_https_callback' ) ) ? 'https' : '';
```

Comment:
Why '' vs null?

Reply:
it is used later `$site_url = empty( $protocol ) ? site_url( 'index.php' ) : site_url( 'index.php', $protocol );`.

Honestly it is the same but would prefer null too

Reply:
@cocojoe @glena - I agree, change made!